airgun/widgets.py

```diff
@@ -1321,6 +1321,42 @@ class EditableEntryMultiCheckbox(EditableEntry):
     edit_field = CheckboxGroup(locator='.//form')
 
 
+class EditableEntryMultiTextInput(GenericLocatorWidget):
```

Comment:
TextInput and EditableEntry can  considered to have the same meaning here, perhaps
LabeledTextInputs
Or TextInputsGroup ...

Reply:
Is this just about the class name? If yes then OK, why not.